test/testutil/driver.c

```diff
@@ -280,20 +280,20 @@ void set_test_title(const char *title)
     test_title = title == NULL ? NULL : strdup(title);
 }
 
-PRINTF_FORMAT(2, 3) static void test_verdict(int pass, const char *extra, ...)
+PRINTF_FORMAT(2, 3) static void test_verdict(int verdict,
+                                             const char *extra, ...)
 {
     va_list ap;
 
     test_flush_stdout();
     test_flush_stderr();
 
-    test_printf_stdout("%*s%s", level, "", pass ? "ok" : "not ok");
-    if (extra != NULL) {
-        test_printf_stdout(" ");
-        va_start(ap, extra);
-        test_vprintf_stdout(extra, ap);
-        va_end(ap);
-    }
+    test_printf_stdout("%*s%s ", level, "", verdict ? "ok" : "not ok");
+    va_start(ap, extra);
+    test_vprintf_stdout(extra, ap);
```

Comment:
The NULL check for `extra` was removed, if it is NULL, this will crash. `_dopr()` in b_print.c does not do any NULL checks for the format argument.

Reply:
It's a local function that is never called with extra == NULL.  It also doesn't make much sense to not pass something in.  I'll rename the argument to _description_.